src/app.ts

```diff
@@ -67,25 +74,55 @@ app.get("/map", (req, res) => {
     return [date, mapOwn(mapByState, (val, state) => [state, val])];
   });
   let maxValue = 0;
-  data.forEach(entry =>
-    entry[1].forEach(state => (maxValue = Math.max(maxValue, state[1])))
+  data.forEach((entry) =>
+    entry[1].forEach((state) => (maxValue = Math.max(maxValue, state[1])))
   );
   res.send({
     data,
     maxValue,
   } as MapData);
 });
 
+app.get("/range-timeseries-data", (req, res) => {
+  let keys:string[] = Object.keys(db);
+  const dataForAllInterventions =  keys.map((key) => {
+    let dataToProcess = db[key];
+    const rowsGroupedByDate: Map<number, Entry[]> = new Map<number, Entry[]>();
+    const stateCode: string = req.query.stateCode;
+    if (stateCode != null) {
+      dataToProcess = dataToProcess.filter((row) =>
+        row.county.endsWith(stateCode)
+      );
+    }
+
+    dataToProcess.forEach((row) => {
+      const timestamp = moment(row.Date, "MM/DD/YYYY").unix() * 1000;
```

Comment:
it's unnecessary to use moment.js just for turning it to a timestamp. `(new Date('3/4/20')).getTime()`